forms-flow-api/src/api/resources/checkpoint.py

```diff
@@ -18,10 +14,9 @@ class HealthCheckpointResource(Resource):
     """Resource for managing healthcheckpoint."""
 
     @staticmethod
-    @cors.crossdomain(origin=CORS_ORIGINS, max_age=21600)
     def get():
         """Get the status of API."""
         return (
-            jsonify({"message": "Welcome to formsflow.ai API"}),
+            ({"message": "Welcome to formsflow.ai API"}),
```

Comment:
Need to add these messages to message bundles later

Reply:
Yeah for internationalization right?